scripter.py

```diff
@@ -38,6 +38,8 @@ def clear_terminal():
 
 def change_terminal(image_file_path):
```

Comment:
Sorry for all the breakage!

While here, please consider adding the following sanity checks as the first line of both `change_terminal()` and `change_wallpaper()`.

```python
def change_terminal(image_file_path):
    assert isinstance(image_file_path, str), 'image_file_path must be a str'
# [ ... ]
def change_wallpaper(image_file_path):
    assert isinstance(image_file_path, str), 'image_file_path must be a str'
```